SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealObjectRef.h

```diff
@@ -85,6 +69,9 @@ struct FUnrealObjectRef
 		return (*this != NULL_OBJECT_REF && *this != UNRESOLVED_OBJECT_REF);
 	}
 
+	UObject* ToObjectPtr(USpatialPackageMapClient* PackageMap, bool& bOutUnresolved) const;
+	static FUnrealObjectRef FromObjectPtr(UObject* ObjectValue, USpatialPackageMapClient* PackageMap);
```

Comment:
Nit: it seems the arguments can be const (same for the ToObjectPtr)?
static FUnrealObjectRef FromObjectPtr(const UObject* ObjectValue, const USpatialPackageMapClient* PackageMap);

Reply:
PackageMap can get modified when resolving something new (e.g. for stably named objects). `ObjectValue` can't be const either because it's passed as non-const to the `PackageMap` functions.

Reply:
I see, all right.